ethexe/signer/src/lib.rs

```diff
@@ -356,7 +312,7 @@ mod tests {
         let hash = keccak256(message);
 
         // Recover the address using the signature
-        let ethers_sig = ethers::core::types::Signature::try_from(&signature.0[..])
+        let ethers_sig = ethers::core::types::Signature::try_from(signature.as_ref())
```

Comment:
it would be nice to refactor this and remove the deprecated ethers crate in favor of alloy

Reply:
Done